### PR TITLE
Contentpackstats

### PR DESCRIPTION
**Pull request by jamalex**
## Summary

Simple management command to display a few stats about a currently installed language pack, to help with debugging/checking.

## Screenshots (if appropriate)

```
$ bin/kalite manage contentpackstats es-ES

Number of videos: 131
Number of exercises: 1333
Number of topics: 1628
Number of khan-exercises files: 30
Number of subtitle files: 110

Metadata:
	video_count: 0
	code: es-ES
	topic_tree_translated: 74.4869361129
	name: Spanish, Spain
	software_version: 0.16
	subtitle_count: 110
	language_pack_version: 1
	native_name: Español
	percent_translated: 85.8064516129
```

### CODE REVIEW

**Comment by jamalex** [comment]
(This is based off of 0.16.x, which doesn't actually seem to have been merged to develop too recently, so the merge conflict should be resolved there first)

File: kalite/distributed/management/commands/contentpackstats.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import ugettext as _
+
+from fle_utils.general import ensure_dir
+
+from kalite.topic_tools.content_models import get_content_items, Item, set_database, parse_data
+
+from kalite.i18n.base import lcode_to_django_lang, get_po_filepath, get_locale_path, \
+    download_content_pack, update_jsi18n_file, get_subtitle_file_path as get_subtitle_path, \
+    extract_content_db, get_localized_exercise_dirpath
```
Comment (benjaoming) [inline kalite/distributed/management/commands/contentpackstats.py]: Unused imports galore? :) Do you have a static checker in your IDE?

**Comment by benjaoming** [comment]
@jamalex this looks very useful!

Could it be run without the language code? Because one thing to troubleshoot would be *which language packs are installed*, and typically you get in doubt about what the correct language code is :)

Another possible addition: Just change the command to be called `stats` ?

`kalite manage stats`: Outputs statistics for all languages available. Would be a very nifty shortcut :)

**Comment by benjaoming** [comment]
@jamalex 0.16.x is now merged into develop, so you should be able to `git rebase develop` and `git push -f` on your feature branch.

**Comment by jamalex** [comment]
Rebased.

**Comment by benjaoming** [comment]
Hurray new features :)

**Pull request merged**